index.js

```diff
@@ -759,3 +759,20 @@ is.hex = function (value) {
 is.symbol = function (value) {
   return typeof Symbol === 'function' && toStr.call(value) === '[object Symbol]' && typeof symbolValueOf.call(value) === 'symbol';
 };
+
+is.oneOf = function (value) {
+  for (var _len = arguments.length, types = Array(_len > 1 ? _len - 1 : 0), _key = 1; _key < _len; _key++) {
```

Comment:
In the latter case, wouldn't you want to `return value && value.length || null`?

If you want to make the changes I suggested above (remove the variadism) I'll definitely be willing to add this.